soroban-sdk/src/env.rs

```diff
@@ -254,15 +254,28 @@ struct EnvTestState {
 pub struct EnvTestConfig {
     /// Capture a test snapshot when the Env is dropped, causing a test snapshot
     /// JSON file to be written to disk when the Env is no longer referenced.
-    /// Defaults to true.
+    /// Defaults to `true`.
     pub capture_snapshot_at_drop: bool,
+    /// When `true` enables detailed per-invocation resource and fee estimation,
+    /// accessible via `env.cost_estimate()`.
+    ///
+    /// The top-level contract invocations and lifecycle operations (such as
+    /// `register` or `env.deployer()` operations) will be metered and
+    /// all the metering information will reset in-between them (note, that this
+    /// includes the CPU budget). Metering will not be reset while inside the
+    /// call (e.g. if a contract calls or creates another contract, that won't
+    /// reset metering).
+    ///     
+    /// Defaults to `false`.
+    pub enable_invocation_cost_estimate: bool,
```

Comment:
I think this can be shortened to:
```suggestion
    pub enable_cost_estimate: bool,
```
The average developer won't distinguish between an invocation cost estimate vs another type of cost estimate.